modules/phpinfo/src/phpinfo.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
Example of how to use a traditional PHP script. As mentioned in ScriptResponse, $this is available (instanceof ScriptResponse) and the file can `return` its own response for chaining.